metadata-ingestion/src/datahub/ingestion/source/sql/oracle.py

```diff
@@ -184,8 +579,16 @@ def get_inspectors(self) -> Iterable[Inspector]:
             event.listen(
                 inspector.engine, "before_cursor_execute", before_cursor_execute
             )
-            # To silent the mypy lint error
-            yield cast(Inspector, OracleInspectorObjectWrapper(inspector))
+            logger.info(
+                f'Data dictionary mode is: "{self.config.data_dictionary_mode}".'
+            )
+            # Sqlalchemy inspector uses ALL_* tables as per oracle dialect implementation.
+            # OracleInspectorObjectWrapper provides alternate implementation using DBA_* tables.
+            if self.config.data_dictionary_mode != "ALL":
+                yield cast(Inspector, OracleInspectorObjectWrapper(inspector))
```

Comment:
Wondering, if we can rename `OracleInspectorObjectWrapper` to something like `OracleDBAInspectorObjectWrapper`, to make this more explicit in class name itself.

Reply:
Oracle allows three levels of access (based on levels of privilege): ALL, DBA, USER. Right now ALL mode is using as default. DBA mode is implemented by this PR. But pottentially, mode USER can be implemented as future contribution. In this case renaming of this class will not give nothing. I can extend docstring, if it is necessary.

Reply:
I see.